e2e/package.json

```diff
@@ -11,7 +11,8 @@
   "devDependencies": {
     "@testing-library/cypress": "^7.0.5",
     "axe-core": "^4.3.0",
-    "cypress": "^6.9.1"
+    "cypress": "^6.9.1",
+    "cypress-plugin-tab": "^1.0.5"
```

Comment:
Cypress maintainers feel adding tab press functionality is out of scope, so added plugin to enable tab navigation.  See: https://github.com/cypress-io/cypress/issues/299